mattlianje/gregarious-wolf
# Audio analysis refactor

Moved helper classes to lib, added location for audio files, added secrets.py, added our rds db url 

**TESTING INSTRUCTIONS:**

1. `$ cd audio-analysis`
2. Install a MySQL client (I am using DataGrips, but I guess MySQL workbench is fine)
3. Ask me for the `secrets.py` and copy it into the `audio-analysis` directory
4. Make sure `test_table` in your `climactic_test` DB has been deleted
5. `$ python main.py log`, then pick option `3`
6. Add the rds url to your MySQL client (like below) and you are good to query.
![image](https://user-images.githubusercontent.com/20251759/73131523-6188d800-3fda-11ea-847b-10517f3a251c.png)


**TO DO:**
Set up read-only users for db.
